docs/fastdds/discovery/discovery_server.rst

```diff
@@ -105,22 +102,24 @@ section <DS_DiscoverySettings>`. The examples below show how to manage the corre
 |    :language: xml                                                                                                    |
 |    :start-after: <!-->CONF-SERVER-DISCOVERY-PROTOCOL<-->                                                             |
 |    :end-before: <!--><-->                                                                                            |
+|    :lines: 2-3,5-17                                                                                                  |
+|    :append: </profiles>                                                                                              |
 +----------------------------------------------------------------------------------------------------------------------+
 
 .. _DS_guidPrefx:
 
 The GuidPrefix as the server unique identifier
-""""""""""""""""""""""""""""""""""""""""""""""
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 The |GuidPrefix_t-api| attribute belongs to the RTPS specification and univocally identifies each RTPSParticipant.
-It consists on 12 bytes and in Fast DDS is a key for the DomainParticipant used in the DDS domain.
+It consists on 12 bytes, and in Fast DDS is a key for the DomainParticipant used in the DDS domain.
 Fast DDS defines the DomainParticipant |GuidPrefix_t-api| as a public data member of the
 |WireProtocolConfigQos-api| class.
 In the Discovery Server, it has the purpose to link a *server* to its *clients*.
-It must be mandatorily specified in: *server* and *client* setups.
+It must be specified in *server* and *client* setups.
 
 Server side setup
-*****************
+"""""""""""""""""
 
 The examples below show how to manage the corresponding enum data member and XML tag.
 
```

Comment:
Please, take into account the previous comment about `std::ostream` in the C++ Option 2 option below.